tests/ImageSharp.Tests/project.json

```diff
@@ -20,9 +20,24 @@
     }
```

Comment:
Definitely keep the tests together. I want to throw Stylecop at them though so they are properly formatted and annotated.